app/presenters/check_answers/evidence_uploads_card.rb

```diff
@@ -2,9 +2,22 @@
 
 module CheckAnswers
   class EvidenceUploadsCard < Base
-    def initialize(_claim)
+    attr_reader :evidence_list
+
+    def initialize(claim)
       @group = 'supporting_evidence'
-      @section = 'firm_details'
+      @section = 'supporting_evidence'
+      @evidence_list = SupportingEvidence.where(claim_id: claim.id)
```

Comment:
```
@evidence_list = claim.supporting_evidence
```

Reply:
Tried already - doing this leads to the following error, though it should work: 
`undefined method `supporting_evidence' for #<Claim id: "bbae7d05-c879-4cd8-bd07-3c919daeb6d0", ufn: "12123", office_code: "1A123B", status: "completed"..
`

Reply:
This is one change i didnt make initially. I think we still need to fix the has_many relationship in the claims model for this to work